### PR TITLE
chore: fix karma-vite configuration

### PR DESCRIPTION
**Pull request by Lodin**
This PR fixes Karma / Vite configuration to avoid broken CI

### CODE REVIEW

**Comment by Artur-** [comment]
Which part is fixing the problem?

**Comment by Lodin** [comment]
@Artur-, moving the Vite config to ge karma config, and some minor updates for it

File: .eslintrc [hunk 1]
```diff
@@ -9,6 +9,7 @@
   "plugins": ["tsdoc"],
   "rules": {
     "@typescript-eslint/member-ordering": "off",
+    "@typescript-eslint/return-await": "off",
```
Comment (Artur-) [inline .eslintrc:12]: Why do we want this and how is it related?

Comment (Lodin) [reply .eslintrc:12]: It looks like a leftover from one of my attempts 🤔 We can remove it for now

Comment (Lodin) [reply .eslintrc]: Done

File: packages/ts/react-form/test/useForm.spec.tsx [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 import sinonChai from 'sinon-chai';
 import { useForm as _useForm, useFormPart } from '../src/index.js';
-import { type Login, LoginModel, UserModel } from './models.js';
+import { type Login, LoginModel, type UserModel } from './models.js';
 
 use(sinonChai);
+use(chaiDom);
+
+describe('@hilla/react-form', function () {
+  this.timeout(100000);
```
Comment (platosha) [inline packages/ts/react-form/test/useForm.spec.tsx:14]: I guess this is leftover from debugging, let's clean it up.

```suggestion
```

File: scripts/build.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ await build({
   },
   entryPoints: srcFiles.map((file) => new URL(file, root)).map(fileURLToPath),
   format: 'esm',
-  minify: true,
+  // minify: true,
```
Comment (platosha) [inline scripts/build.ts:22]: I'm curious, what was the reason for disabling this? Was minified code problematic for debugging?

**Review by platosha — CHANGES_REQUESTED** [review]

File: scripts/build.ts (hunk 3 above)
Comment (Lodin) [reply scripts/build.ts:22]: This is a debug leftover. Though, the minification causes Vite to hiccup on sourcemaps for some reason, so all the lines are messed up. That's why I'm considering disabling this option until this bug is fixed. What do you think?

File: package.json [hunk 4]
```diff
@@ -30,7 +30,11 @@
   "license": "Apache-2.0",
   "overrides": {
     "esbuild": "$esbuild",
-    "prettier": "$prettier"
+    "prettier": "$prettier",
+    "@vaadin/vaadin-lumo-styles": "24.2.0-rc1",
```
[the remark this answers is not in the document]
Comment (Lodin) [reply package.json:34]: Good notice. There was some `dev` versions of the package left across web components. Let's check if it is correct now and file an issue if not.

File: scripts/build.ts (hunk 3 above)
Comment (platosha) [reply scripts/build.ts]: > This is a debug leftover. Though, the minification causes Vite to hiccup on sourcemaps for some reason, so all the lines are messed up. That's why I'm considering disabling this option until this bug is fixed. What do you think?

Agree. Also bug reports with real thing names in the error messages and stack traces are much nicer than what we have now with mangled names.

Comment (platosha) [reply scripts/build.ts]: It's fine to address minification disabling in a separate PR though.

Comment (Lodin) [reply scripts/build.ts]: Yep, will do a separate one

Comment (Lodin) [reply scripts/build.ts]: Btw, mangled names are a bug I believe. I know why it happened and it would be better to fix it eventually.

**Review by platosha — APPROVED** [review]

**Pull request merged**